themes/default/views/administrate/setup/Results/docx_results.php

```diff
@@ -129,7 +128,7 @@
 		$contentCell = $table->addCell(12 * $cmToTwips);
 
 		$contentCell->addText(
-			html_entity_decode(strip_tags(br2nl($vo_result->get('preferred_labels'))), ENT_QUOTES | ENT_HTML5),
+			htmlentities(html_entity_decode(strip_tags(br2nl($vo_result->get('preferred_labels'))), ENT_QUOTES | ENT_HTML5), ENT_XML1),
```

Comment:
All text added via `addText()`  are  now wrapped in `htmlentities($text, ENT_XML1)`